.travis.yml

```diff
@@ -0,0 +1,65 @@
+language: android
+jdk: oraclejdk8
+sudo: required
```

Comment:
Is sudo actually needed? As far as I see, you do not use sudo or adp. etc. Not requiring sudo would make the Travis CI run much faster.

Reply:
I think the reason why I added this is because if `sudo` is required, the build process happens in a different environment: see also https://docs.travis-ci.com/user/reference/overview/
I think I had some problems when not requiring sudo...

Reply:
I did another test without `sudo` and it took as long as with `sudo` and also exited at the same point because of the same reason: https://travis-ci.org/ENT8R/StreetComplete/jobs/398196246#L3118
So there is no huge time difference: https://travis-ci.org/ENT8R/StreetComplete/builds
Do you think I should remove it?

Reply:
Okay, don't know the end results in timing, by theoretical at least when multiple runs are executed it may be faster.

Without sudo it runs in a "lightweight" container at Travis, with sudo it uses a full VM. That's why I would always prefer it without sudo if you don't need it.